docs/UserGuide.adoc

```diff
@@ -304,8 +307,9 @@ Note: The username and password are both case-sensitive.
 The default account can be accessed with `login user/admin pw/passw0rd`
 
 A login allows the user to access the commands which can affect the
-bookings. Commands which require login: `add`, `checkin`, `checkout`, `reassign`,
-`service` and `clear`.
+bookings. +
+Commands which require login: `add`, `checkin`, `checkout`,
+`reassign`, `service` and `clear`.
```

Comment:
archive/export also requires login. Unless you don't want to include it because it is only gonna be seen in v2.0